src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -280,10 +280,7 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
                 try
                 {
                     // open the input file
-                    using (FileStream fs = new FileStream(InFile, FileMode.Open))
-                    {
-                        SetRequestContent(request, fs);
-                    }
+                    SetRequestContent(request, new FileStream(InFile, FileMode.Open));
```

Comment:
Why did you remove the using statement here? The original code looks more accurate